packages/react-components/src/components/GridLayout.tsx

```diff
@@ -31,43 +24,99 @@ export interface GridLayoutProps {
   styles?: BaseCustomStylesProps;
 }
 
-const calculateStandardLayoutRows = (numberOfItems: number, gridCol: number): number =>
-  Math.ceil(numberOfItems / gridCol);
-
-const calculateStandardLayoutColumns = (numberOfItems: number): number =>
-  numberOfItems > 0 ? Math.ceil(Math.sqrt(numberOfItems)) : 1;
-
 /**
- * A component to lay out audio / video tiles in a call.
+ * A component to lay out audio / video participants tiles in a call.
  *
  * @public
  */
 export const GridLayout = (props: GridLayoutProps): JSX.Element => {
-  const [gridCol, setGridCol] = useState(1);
-  const [gridRow, setGridRow] = useState(1);
-
-  const { children, layout = 'standard', styles } = props;
+  const { children, styles } = props;
   const numberOfChildren = React.Children.count(children);
 
-  switch (layout) {
-    case 'standard': {
-      const numberOfColumns = calculateStandardLayoutColumns(numberOfChildren);
-      if (gridCol !== numberOfColumns) setGridCol(numberOfColumns);
-      const numberOfRows = calculateStandardLayoutRows(numberOfChildren, gridCol);
-      if (gridRow !== numberOfRows) setGridRow(numberOfRows);
-      break;
+  const targetRef = useRef<HTMLDivElement>(null);
+  const [blockProps, setBlockProps] = useState<BlockProps>({
```

Comment:
nit: The name `blockProps` makes me think that Block is a component and these are its props.
Can we use something like `calculatedBlocks` etc?